include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -301,23 +301,16 @@
 #define _ONEDPL_CPP20_SHIFT_LEFT_RIGHT_PRESENT                                                                         \
     (_ONEDPL___cplusplus >= 202002L && (_MSC_VER >= 1921 || _GLIBCXX_RELEASE >= 10))
 
-#if _ONEDPL___cplusplus >= 202002L && __cpp_lib_ranges >= 201911L
-#   define _ONEDPL_CPP20_RANGES_PRESENT 1
-#else
-#    define _ONEDPL_CPP20_RANGES_PRESENT 0
-#endif
-
-// Feature public macros
-#define ONEDPL_HAS_RANGES_API _ONEDPL_CPP20_RANGES_PRESENT
-
 #if _ONEDPL_STD_FEATURE_MACROS_PRESENT
 #    define _ONEDPL_CPP20_SPAN_PRESENT (__cpp_lib_span >= 202002L)
 #    define _ONEDPL_CPP20_CONCEPTS_PRESENT (__cpp_concepts >= 201907L && __cpp_lib_concepts >= 202002L)
+#    define _ONEDPL_CPP20_RANGES_PRESENT 1
```

Comment:
```suggestion
#    define _ONEDPL_CPP20_RANGES_PRESENT (_ONEDPL___cplusplus >= 202002L && __cpp_lib_ranges >= 201911L)
```

Reply:
Since `_ONEDPL_STD_FEATURE_MACROS_PRESENT` depends on `_ONEDPL___cplusplus`, I do not mind against `#    define _ONEDPL_CPP20_RANGES_PRESENT (__cpp_lib_ranges >= 201911L)`